### PR TITLE
[ADD] account_brand

### PR DESCRIPTION
**Pull request by osi-scampbell**
#564 

Depends on:
- [x] https://github.com/OCA/partner-contact/pull/755

### CODE REVIEW

**Comment by max3903** [comment]
Ping @RLeeOSI

File: account_brand/views/account_invoice.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<odoo>
+
+    <!-- Copyright (C) 2019 Open Source Integrators
+         License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+    <record id="invoice_form_brand" model="ir.ui.view">
```
Comment (PieterPaulussen) [inline account_brand/views/account_invoice.xml]: Please align with the [desired standard for view naming](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action)

**Review by max3903 — APPROVED** [review]

**Review by osimallen — APPROVED** [review]
Looks good

**Pull request merged**